wallaby.js

```diff
@@ -1,14 +1,23 @@
-module.exports = function (wallaby) {
+module.exports = function(wallaby) {
     return {
+        env: {
```

Comment:
This may not be required, but wanted to highlight that we are running these tests as node and not browser tests. This would change if we wanted to run browser specific tests AND we were using Mocha (as opposed to Jest which is a node runner with jsdom only).